frontend/src/components/auth/Signup.jsx

```diff
@@ -31,11 +31,12 @@ const Signup = () => {
                 await setDoc(doc(db, "users", userid), {
                     username: username,
                     email: email,
-                    folderNames: [`${username}'s Default Folder`]
+                    folderNames: [`${username}'s Default Folder`], 
+                    transcripts: {}
                 });
 
-                const foldersRef = collection(db,"users", userid, `${username}'s Default Folder`);
-                await addDoc(foldersRef,{
+                const foldersRef = doc(db,"users", userid, `${username}'s Default Folder`,'initial');
```

Comment:
thoughts on not showing the "initial" file to the user?

Reply:
It's a blank file, purely used to initialize the collection